system/install.py

```diff
@@ -221,251 +221,265 @@ def systemInstall():
+                print("ERROR: Unable to build system workers ")
+                return False
+            if not jimi.settings._settings().new("cpuSaver",config["cpuSaver"]):
+                print("ERROR: Unable to build system cpuSaver ")
+                return False
+            if not jimi.settings._settings().new("scheduler",config["scheduler"]):
+                print("ERROR: Unable to build system scheduler ")
+                return False
+            if not jimi.settings._settings().new("cluster",config["cluster"]):
+                print("ERROR: Unable to build system cluster ")
+                return False
+            if not jimi.settings._settings().new("audit",config["audit"]):
+                print("ERROR: Unable to build system audit ")
+                return False
+            if not jimi.settings._settings().new("auth",config["auth"]):
+                print("ERROR: Unable to build system auth ")
+                return False
+            if not jimi.settings._settings().new("cache",{ "garbageCollector" : True }):
+                print("ERROR: Unable to build system cache ")
+                return False
+            sys.exit(0)
+
+    if currentVersion < 3.01:
+        jimi.model.registerModel("editorUI","_editorUI","_document","core.models.webui")
+
+    if currentVersion < 3.02:
+        # Adding default everyone group
+        everyoneGroup = jimi.auth._group().getAsClass(query={ "name" : "everyone" })
+        if len(everyoneGroup) == 0:
+            everyoneGroup = jimi.auth._group().new("everyone")
+        # Update ACLs of editorUI model for each object to fix bug
+        existingObjects = jimi.webui._editorUI().getAsClass()
+        for existingObject in existingObjects:
+            existingObject.acl = {"ids":[{"accessID":0,"delete":True,"read":True,"write":True}]}
+            existingObject.update(["acl"])
+
+    if currentVersion < 3.03:
+        jimi.model.registerModel("extract","_extract","_action","system.models.extract")
+
+    if currentVersion < 3.034:
+        loadSystemManifest()
+
+    if currentVersion < 3.035:
+        jimi.model.registerModel("revision","_revision","_document","core.revision")
+
+    if currentVersion < 3.036:
+        users = jimi.auth._user().getAsClass(query={  })
+        for user in users:
+            user.whatsNew = True
+            user.update(["whatsNew"])
+
+    if currentVersion < 3.04:
+        #Adding additional auth settings
+        authSettings = jimi.settings._settings().getAsClass(query={"name" : "auth"})[0]
+        if "types" not in authSettings.values:
+            authSettings.values["types"] = ["local"]
+            authSettings.update(["values"])
+
+        #Add ldap and oauth settings
+        jimi.settings._settings().new("ldap",{"domains":[]})
+        jimi.settings._settings().new("oauth",{})
+
+        #Adding org model
+        jimi.model.registerModel("organisation","_organisation","_document","core.organisation")
```

Comment:
Do we include this within install as well for new system build?

Reply:
Added this into the standard system install